controllers/users.js

```diff
@@ -11,7 +11,7 @@ function create(req, res, next) {
   const { name, email, password } = req.body
 
   User.create({ name, email, password })
-    .then((user) => res.json(user))
+    .then((user) => res.status(201).json(user))
     .catch((err) => next(err))
```

Comment:
This part could use a refactoring by changing the `.catch` line to just `.catch(next)`.